lib/streamlit/elements/media.py

```diff
@@ -133,31 +136,42 @@ def video(
 
         Parameters
         ----------
-        data : str, bytes, BytesIO, numpy.ndarray, or file
+        data : str, bytes, io.BytesIO, numpy.ndarray, or file
             Raw video data, filename, or URL pointing to a video to load.
             Includes support for YouTube URLs.
             Numpy arrays and raw data formats must include all necessary file
             headers to match specified file format.
+
         format : str
-            The mime type for the video file. Defaults to 'video/mp4'.
+            The mime type for the video file. Defaults to ``"video/mp4"``.
             See https://tools.ietf.org/html/rfc4281 for more info.
+
         start_time: int
             The time from which this element should start playing.
-        subtitles: str, dict, or io.BytesIO
+
+        subtitles: str, bytes, Path, io.BytesIO, or dict
             Optional subtitle data for the video, supporting several input types:
-            * None (default): No subtitles.
-            * A string: File path to a subtitle file in '.vtt' or '.srt' formats, or
+
+            * ``None`` (default): No subtitles.
+
+            * A string, bytes, or Path: File path to a subtitle file in ``.vtt`` or ``.srt`` formats, or
               the raw content of subtitles conforming to these formats.
               If providing raw content, the string must adhere to the WebVTT or SRT
               format specifications.
+
+            * io.BytesIO: A BytesIO stream that contains valid ``.vtt`` or ``.srt``
+              formatted subtitle data.
+
             * A dictionary: Pairs of labels and file paths or raw subtitle content in
-              '.vtt' or '.srt' formats.
-              Enables multiple subtitle tracks. The label will be shown in the video
-              player. Example:
-              {'English': 'path/to/english.vtt', 'French': 'path/to/french.srt'}
-            * io.BytesIO: A BytesIO stream that contains valid '.vtt' or '.srt'
-              formatted subtitle data. When provided, subtitles are displayed
-              by default. For multiple tracks, the first one is displayed by default.
+              ``.vtt`` or ``.srt`` formats to enable multiple subtitle tracks.
+              The label will be shown in the video player. Example:
+              ``{"English": "path/to/english.vtt", "French": "path/to/french.srt"}``
+
+            When provided, subtitles are displayed by default. For multiple
+            tracks, the first one is displayed by default. If you don't want any
+            subtitles displayed by default, pass as empty string in the first
```

Comment:
> pass as empty string in the first entry in the dictionary

This doesn't read as gramatically right

Reply:
Sorry. "an" one sec. Thanks.

Reply:
To be even more precise, we may want to say "pass an empty string as the value in the first entry of a dictionary"... Since "None" isn't an empty string and it technically is the first half of the first entry in the dict 😅 But this might be getting too long

Reply:
I had the same thought honestly and waffled on the same point for the same reason.

Reply:
Let's just do it I guess, or else folks might do {"":"", ...}, as I had initially done.

Reply:
I'll update it.

Reply:
>If you don't want any
            subtitles displayed by default, use an empty string for the value
            in a dictrionary's first entry: ``{"None": "", "English": "path/to/english.vtt"}``